megamek/src/megamek/common/Dropship.java

```diff
@@ -82,6 +82,11 @@ public int getASEWAffected(int arc) {
     private boolean kfBoomDamaged = false;
     private int collarType = COLLAR_STANDARD;
 
+    @Override
+    public boolean tracksHeat() {
```

Comment:
Dropships don't track heat?

Reply:
They operate on a heat-neutral basis. They can't generate more heat in a turn than they can dissipate, so they don't track it round-to-round like mechs, asfs, and small craft.

Reply:
Ok. I think that might confuse people in the future, so let's stick a comment on this method explaining it.